### DISCUSSION TITLE
如何单独运行某章内容

### DISCUSSION THREAD

**Discussion by checknullnow** [sjtug/SJTUThesis#702] [提问] [answered]
我现在只会修改完内容然后点compile，生成全文，感觉有点不方便，不知道怎么能单独运行某章内容，来检查是否有误，谢谢

**Comment by skyzh** [comment]
LaTeX 应该不支持这种功能。如果编译速度慢，可以加上 draft 选项。

**Comment by stone-zeng** [comment, marked as answer]
通常的做法是每章分成不同文件，然后用 `\input` 或者 `\include` 插入，检查的时候注释掉不想关的内容。
